### DISCUSSION TITLE
Feature Request:  Client trust account for invoicing

### DISCUSSION THREAD

**Discussion by sollermun** [kimai/kimai#4735] [General]
It would be great if the invoice app would have a feature that allows for trust accounting.

My business takes deposits from clients in advance of work that is performed. When invoices issue, amounts are deducted from the funds held in trust. It would be great if the invoice plugin had a feature for this.

**Comment by kevinpapst** [comment]
You can add negative expenses, which will be deduct from the invoice amount.

Give it a try in the demo system.
